src/ui/views/session/sidebar/SpacesView.tsx

```diff
@@ -54,6 +55,13 @@ export function SpacesView() {
             variant="surface-low"
           />
         </BadgeWrapper>
+        <SidebarTab
+          onClick={() => selectWindow(OverlayWindow.Discover)}
+          isActive={selectedWindow === OverlayWindow.Discover}
+          name="Discover"
+          iconSrc={ExploreIC}
+          variant="danger"
```

Comment:
I think we should just use the same color as the other sidebar tabs here. It made sense in the original designs to separate it from other spaces, but I don't know if it does anymore.